nodes/lite-nodes/spin-up-a-lite-node.md

```diff
@@ -33,18 +33,18 @@ To build the lite-node, you’ll need some specific software. Run the following
 
     ```sh
     sudo apt update -y
-    sudo apt install mesa-opencl-icd ocl-icd-opencl-dev gcc git bzr jq pkg-config curl clang build-essential hwloc libhwloc-dev wget -y
+    sudo apt install mesa-opencl-icd ocl-icd-opencl-dev gcc git jq pkg-config curl clang build-essential hwloc libhwloc-dev wget -y
     ```
 
 2.  [Install Go](https://go.dev/doc/install) and add `/usr/local/go/bin` to your `$PATH` variable:
 
     ```sh
-    wget https://go.dev/dl/go1.21.1.linux-amd64.tar.gz
-    sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go1.21.1.linux-amd64.tar.gz
+    wget https://go.dev/dl/go1.21.7.linux-amd64.tar.gz
+    sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go1.21.7.linux-amd64.tar.gz
     echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc && source ~/.bashrc
```

Comment:
For a while, when I had more time, do a global replace whenever there was a new lotus release, or a go update Should we (outside of github, set up some kind of alert to keep us doing this)

Reply:
There's a workflow over in IPFS Docs that does this. Might be as easy as copying-and-pasting into this repo: https://github.com/ipfs/ipfs-docs/blob/main/.github/workflows/update-on-new-ipfs-tag.yml